### DISCUSSION TITLE
How to train the BasicVSR++ model ofNTIRE 2021 Quality Enhancement of Compressed Video

### DISCUSSION THREAD

**Discussion by StevenTony611** [open-mmlab/mmediting#1281] [How-to] [answered]

**Comment by StevenTony611** [comment]
We cannot find the train_pipline in the config files of the basicvsr++ of the NTIRE 2021 Quality Enhancement. We want to get the training pipeline for NTIRE 2021 Quality Enhancement. Thank you very much!

**Comment by zengyh1900** [comment, marked as answer]
Hi @StevenTony611 

Currently, MMEditing only provides the testing config files for NTIRE 2021.
